df.units.xml

```diff
@@ -385,7 +385,7 @@
                    comment='Seems to be used as default when not flags1.tame'/>
     </enum-type>
 
-    <enum-type type-name='unit_report_type' base-type='int16_t'>
+    <enum-type type-name='unit_report_type' base-type='int16_t' original-name='UnitAnnouncementCategory'>
```

Comment:
sadly `original-name` isn't allowed on `enum-type` (yet) so this gets noted either as a `comment` or just as adjacent XML plaintext